keyboards/rart/rartpad/rartpad.h

```diff
@@ -0,0 +1,41 @@
+
+#pragma once
+
+#include "quantum.h"
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_all( \
+	K00, K01, K02, K03, \
+	K10, K11, K12, K13, \
+	K20, K21, K22, K23, \
+	K30, K31, K32, K33, \
+	K40, K41, K42, K43  \
+) \
+{ \
+	{ K00,   K01,   K02,   K03 }, \
+	{ K10,   K11,   K12,   K13 }, \
+	{ K20,   K21,   K22,   K23 }, \
+	{ K30,   K31,   K32,   K33 }, \
+	{ K40,   K41,   K42,   K43 }, \
+}
+#define LAYOUT_pad( \
+	K00, K01, K02, K03, \
+	K10, K11, K12, K13, \
+	K20, K21, K22,      \
+	K30, K31, K32, K33, \
+	K40,      K42      \
```

Comment:
```suggestion
	K10, K11, K12,      \
	K20, K21, K22, K13, \
	K30, K31, K32,      \
	K40,      K42, K33, \
```

Reply:
i try to fix this but always fail.  where is the error? I have followed the instructions

Reply:
Whoops sorry. Fixed.

Reply:
done

Reply:
dont  know but  always error in LAYOUT_numpad_5x4

Reply:
solved, thanks